cosmic-core/db-scripts/src/main/resources/db/schema-536to537-cleanup.sql

```diff
@@ -2,3 +2,6 @@
 -- Schema cleanup from 5.3.6 to 5.3.7;
 --
 
+DROP TABLE IF EXISTS `cloud`.`snapshot_policy`;
```

Comment:
Looks fine to me. But shouldn't it be on `schema-536to537.sql` instead of `schema-536to537-cleanup.sql`?